src/DotNetWorker/Grpc/GrpcHttpCookies.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Google.Protobuf.Collections;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal class GrpcHttpCookies : HttpCookies
+    {
+        private RepeatedField<RpcHttpCookie> _cookies;
+
+        public GrpcHttpCookies(RepeatedField<RpcHttpCookie> cookies)
+        {
+            _cookies = cookies;
+        }
+
+        public override void Append(string name, string value)
```

Comment:
It's a bit odd... if you look at the remarks for that documentation, it states that the value cannot be `null` (should be an empty string if not set). The exceptions documentation there also states that a null ref exception will be thrown if the value is null 🤷🏼‍♂️

Reply:
@fabiocav - I think from the code that null will become empty string by default: https://github.com/dotnet/runtime/blob/4893645536af64b01884ce5f3c392ad63513d04b/src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs#L669

Found a related PR here: https://github.com/dotnet/corefx/pull/8206 (which resolves [this issue](https://github.com/dotnet/runtime/issues/17156))

And an issue on [the docs](https://github.com/dotnet/runtime/issues/16891) saying it will throw for null, which appears to be resolved but the docs look the same? :|

Reply:
Yeah, looks like a doc issue.